manifests/00_namespace.yaml

```diff
@@ -3,6 +3,6 @@ kind: Namespace
 metadata:
   labels:
     openshift.io/cluster-monitoring: "true"
-  name: openshift-service-catalog-controller-manager-operator
+  name: openshift-service-catalog-removed
```

Comment:
Just double checking that its okay for both controller-manager and apiserver removers to share this same namespace?

Reply:
Yes, a conversation with the powers that be wanted just one namespace. This is intentional.